Documentation/gettingstarted/encryption.rst

```diff
@@ -115,6 +116,31 @@ traffic) can be enabled as well. See :ref:`node_to_node_encryption` below.
 At this point the Cilium managed nodes will be using IPsec for all traffic. For further
 information on Cilium's transparent encryption, see :ref:`ebpf_datapath`.
 
+.. _encryption_ipam_eni:
+
+Encryption with IPAM ENI mode
+==============================
+
+In order to enable encryption on EKS with the IPAM ENI mode, deploy cilium via
+Helm with the following options:
+
+.. parsed-literal::
+
+    helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set ipam.mode=eni \\
+      --set tunnel=disabled \\
+      --set encryption.enabled=true \\
+      --set encryption.nodeEncryption=false
+
+By default, this enables routing via the ``cilium_host`` interface.
+Specifying encryption interfaces with ``--set encryption.interface=ethX`` is not supported.
```

Comment:
We fixed this right? I needed it to make EKS work in my setup when auto-detection did not pick the correct device.

Reply:
This PR was sent prior to your EKS encryption fixes, I'll update this line.

Reply:
Well, we haven't exactly fixed this, have we? The user specified interface config can cause traffic disruption if `BPF_NETWORK` prog that does decryption is not attached to that interface.